app/javascript/src/ActiveDocs/ThreeScaleApiDocs.ts

```diff
@@ -1,9 +1,12 @@
 import SwaggerUI from 'swagger-ui'
+// this is how SwaggerUI imports this function https://github.com/swagger-api/swagger-ui/pull/6208
+import { execute } from 'swagger-client/es/execute'
```

Comment:
If using the swagger.d.ts suggested above:
```suggestion
import { execute } from 'swagger-client'
```

Reply:
OK this is wrong, the import has to be from 'swagger-client/es/execute'.